tests/Core/Content/Product/SalesChannel/Listing/ProductListingRouteTest.php

```diff
@@ -40,4 +41,16 @@ protected function getOriginal()
     {
         return $this->original;
     }
+
+    public function testWillUseOriginalInCaseTheCategoryIdIsTheMainCategory(): void
+    {
+        $salesChannelContextMock = $this->getMockedSalesChannelContext(true, '69');
+        $request = Request::create('http://your-shop.de/some-category');
+        $request->setSession($this->getSessionMock());
+
+        $productRoute = $this->getRoute();
+
+        $this->getOriginal()->expects($this->once())->method('load');
+        $this->call($productRoute, $request, $salesChannelContextMock);
```

Comment:
Please explicitly set the category id so it's more readable, maybe extract it to a variable.